{{cookiecutter.project_name}}/tests/test_my_module.py

```diff
@@ -4,6 +4,20 @@
 """
 import pytest
 from {{ cookiecutter.package_name }} import my_module
+from testfixtures import LogCapture
+
+
+def test_log_capture(caplog):
```

Comment:
Can you try to use https://docs.pytest.org/en/6.2.x/logging.html#caplog-fixture instead of another dep

Reply:
this function was removed